code/parachain/frame/composable-traits/src/staking/lock.rs

```diff
@@ -16,13 +16,52 @@ use crate::time::{DurationSeconds, Timestamp};
 #[scale_info(skip_type_params(MaxDurationPresets))]
```

Comment:
I prefer if we just remove this lock.rs file entirely and keep everything related to staking in a single file